.github/workflows/aws-deploy.yaml

```diff
@@ -40,11 +40,11 @@ jobs:
           aws-access-key-id: ${{ secrets.CI_SERVICE_USER_ACCESS_KEY }}
           aws-secret-access-key: ${{ secrets.CI_SERVICE_USER_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
-          role-to-assume: arn:aws:iam::563295687221:role/NlpSandboxServiceAccount-ServiceRole-H1K0Q1UH2IX7
```

Comment:
I updated these values and the CI secrets.

Reply:
👍